pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -754,30 +1224,96 @@
+      "INFO: The code has run  8.28213254900038 seconds.\n",
+      "INFO: Estimated remaining time:  6.625706039200304 seconds\n",
+      "INFO: =======Iteration Number: 5 =====\n",
+      "INFO: elapsed time: 0.7\n",
+      "INFO: This is the  6 run out of  9 run.\n",
+      "INFO: The code has run  9.025113655000496 seconds.\n",
+      "INFO: Estimated remaining time:  4.512556827500248 seconds\n",
+      "INFO: =======Iteration Number: 6 =====\n",
+      "INFO: elapsed time: 0.9\n",
+      "INFO: This is the  7 run out of  9 run.\n",
+      "INFO: The code has run  9.944960463000825 seconds.\n",
+      "INFO: Estimated remaining time:  2.841417275143093 seconds\n",
+      "INFO: =======Iteration Number: 7 =====\n",
+      "INFO: elapsed time: 0.7\n",
+      "INFO: This is the  8 run out of  9 run.\n",
+      "INFO: The code has run  10.693073769000875 seconds.\n",
+      "INFO: Estimated remaining time:  1.3366342211251094 seconds\n",
+      "INFO: =======Iteration Number: 8 =====\n",
+      "INFO: elapsed time: 0.9\n",
+      "INFO: This is the  9 run out of  9 run.\n",
+      "INFO: The code has run  11.565678280000611 seconds.\n",
+      "INFO: Estimated remaining time:  0.0 seconds\n",
+      "INFO: =======Iteration Number: 9 =====\n",
+      "INFO: elapsed time: 1.2\n",
+      "INFO: This is the  10 run out of  9 run.\n",
+      "INFO: The code has run  12.722678293001081 seconds.\n",
+      "INFO: Estimated remaining time:  -1.2722678293001082 seconds\n",
+      "INFO: Overall wall clock time [s]:  12.722678293001081\n"
+     ]
+    }
+   ],
    "source": [
-    "doe_object = DesignOfExperiments(parameter_dict, design_gen,\n",
-    "                                 measure_class, create_model,\n",
-    "                                prior_FIM=prior_pass, discretize_model=disc_for_measure)\n",
-    "\n",
-    "all_fim = doe_object.run_grid_search(design_ranges, \n",
-    "                                     mode=sensi_opt)"
+    "# Creating doe_object using DesignOfExperiments\n",
+    "doe_object = DesignOfExperiments(parameter_dict, # dictionary of parameters\n",
+    "                                 design_gen, # design variable\n",
+    "                                 measure_class, # measurement variable\n",
+    "                                 create_model, # model\n",
+    "                                 prior_FIM=prior_pass, # FIM of prior experiments\n",
+    "                                 discretize_model=disc_for_measure # discretized model\n",
+    "                                )\n",
+    "# Grid search\n",
+    "all_fim = doe_object.run_grid_search(design_ranges, # range of design variables\n",
+    "                                     mode=sensi_opt # solver option for sensitivity\n",
+    "                                    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 7: Results through Sensitivity Curves and Heat Maps"
```

Comment:
Full disclosure that I am not a SME, but what does this section add? Why would someone want to use a sensitivity curve and/or heat map? (Also, note, you interchange "heat map" and "heatmap" in several places)